src/services/itemMembership/service.ts

```diff
@@ -117,17 +127,28 @@ export class ItemMembershipService {
   async post(
     actor: Actor,
     repositories: Repositories,
-    { permission, itemId, memberId }: { permission: PermissionLevel; itemId: UUID; memberId: UUID },
+    im: { permission: PermissionLevel; itemId: UUID; memberId: UUID },
```

Comment:
I got confused while reading
```suggestion
    membership: { permission: PermissionLevel; itemId: UUID; memberId: UUID },
```